recipes-dotnet/corehost/coreclr_2.0-beta.bb

```diff
@@ -54,7 +53,18 @@ do_install() {
 	install -m 0755 ${src}/mscorlib.dll ${target}
 	install -m 0755 ${src}/System.Private.CoreLib.dll ${target}
 	install -m 0755 ${src}/SOS.NETCore.dll ${target}
+
+	# Create dev package
+	install -d ${D}/opt/dotnet-dev/
+	for i in `ls ${src}/.nuget/pkg/*.nupkg`
+	do
+		install -m 644 ${i} ${D}/opt/dotnet-dev/
```

Comment:
I think "nuget" would be a more fitting name for that folder since it's already used as a package source. In the future, it could also hold e.g. a self-built libuv package, or the corefx packages.